### ISSUE TITLE
Support for Event Hub Namespace minimum tls settings

### ISSUE THREAD

**Issue by gareda** [hashicorp/terraform-provider-azurerm#17030]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Support for Event Hub Namespace minimum tls settings

### New or Affected Resource(s)/Data Source(s)

azurerm_eventhub_namespace

### Potential Terraform Configuration

```hcl
resource "azurerm_eventhub_namespace" "example" {
  name                = "example-namespace"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  sku                 = "Standard"
  capacity            = 2
  min_tls_version     = "TLS1_2"

  tags = {
    environment = "Production"
  }
}
```

### References

https://docs.microsoft.com/en-us/azure/event-hubs/transport-layer-security-enforce-minimum-version?WT.mc_id=Portal-Microsoft_Azure_EventHub

**Comment by jayendranarumugam** [comment]
Looks like this was available from [2022-01-01-preview](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/namespaces-preview.json#L706-L717). Probably SKD needs to get updates as well

**Comment by davidkarlsen** [comment]
Seems to be supported https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/eventhub_namespace#minimum_tls_version - should this issue be closed?

**Comment by jayendranarumugam** [comment]
Looks like this was released as part of https://github.com/hashicorp/terraform-provider-azurerm/pull/17194 which can be supported from the version [3.17.0](https://github.com/hashicorp/terraform-provider-azurerm/blob/v3.17.0/CHANGELOG.md)

**Comment by rcskosir** [comment]
Thanks for taking the time to submit this issue. It looks like this has been resolved as of #17194. As such, I am going to mark this issue as closed. If that is not the case, please provide additional information including the version in which you are still experiencing this issue, thanks!
